frontend/app/src/components/calendar/CalendarEventList.vue

```diff
@@ -49,14 +49,15 @@ const { t } = useI18n();
         v-if="event.description"
         :popper="{ placement: 'left' }"
         :disabled="!showTooltip"
+        :open-delay="400"
         tooltip-class="max-w-[20rem] whitespace-break-spaces"
       >
         <template #activator>
           <div class="text-body-2 text-rui-text-secondary">
             {{ description }}
           </div>
         </template>
-        {{ event.description }}
+        <span>{{ event.description }}</span>
```

Comment:
does the extra span make a difference here?

Reply:
without the span, it's like there is some space in front of the content (because i use whitespace-break-spaces, but I need it to show the <br> in the content)